src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -8,28 +8,27 @@
 import frc.robot.Constants;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import frc.robot.common.hardware.MotorController;
+//import edu.wpi.first.wpilibj.DigitalInput;
 
 /** Add your docs here. */
 public class CDSSubsystem extends SubsystemBase {
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
   private MotorController CDSBeltController;
+  private MotorController CDSAlignmentWheelOne;
```

Comment:
Private variables should be named m_name.

Reply:
Google Java Style guide doesn't use prefixes:

[5.1 Rules common to all identifiers](https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names)
"In Google Style, special prefixes or suffixes are not used. For example, these names are not Google Style: name_, mName, s_name and kName."

I noted this in #26 which is why they are being removed here.